tensorflow/core/transforms/remapper/pass.cc

```diff
@@ -306,6 +312,249 @@ class BasePatternActivationRewriter : public BasePatternRewriter {
   }
 };
 
+// NOTE(ezhulenev): See `BatchnormSpatialPersistentEnabled` documentation in the
+// `tensorflow/stream_executor/cuda/cuda_dnn.cc` for details.
```

Comment:
technically it's under `tensorflow/compiler/xla/stream_executor/cuda/cuda_dnn.cc`, could you fix that?